modules/ROOT/pages/promote-app-prod-env-design-center.adoc

```diff
@@ -9,29 +9,29 @@ endif::[]
 
 All Mule applications that you create with Flow Designer are created in the Design environment. When a Mule application that you've created is ready to go live, you can deploy it to another environment, if your organization allows you to do so. The target environment must have enough vCores to accommodate the application; otherwise, deployment will fail.
 
-As you deploy your application, you can change connector configurations and reconnection strategies. You need to do this if there are differences between the Design and target environments that would necessitate such changes.
+As you deploy your application, you can change connector configurations and connections. You need to do this if there are differences between the testing and target environments that would necessitate such changes. For example, if you tested your app in the *Sandbox* environment and now want to deploy it to a production environment, you most likely need to replace the connections that each connector used for connecting to test systems with connections for connecting to production systems.
 
-== Procedure
-. Click *Deploy* in the top-right corner of Flow Designer.
+After you deploy your application, the values for all of its properties are secured in Runtime Manager in the *Properties* section of the *Settings* page for the application. To see the list of an application's properties in Runtime Manager, click *Applications*, select the application, click *Settings*, and click *Properties*.
```

Comment:
I think we can specify a bit the fact that property names are shown and property values are hidden in that view.
Maybe we can also reference the CH docs on this
https://docs.mulesoft.com/runtime-manager/secure-application-properties

Reply:
I think it's obvious that the names of the properties appear. Otherwise, how would users know whether the right values are secure?
However, I have added a link to that CH file that you mentioned.